ev3dev-photo-booth.py

```diff
@@ -313,12 +335,15 @@ def _take_picture(self):
         args += ' --scale {0}x{1}'.format(self._fb.resolution.x, self._fb.resolution.y)
         if self._fb.grayscale or self._fb.bits_per_pixel == 1:
             args += ' --greyscale'
-        args += ' --png --save raw.png'
+        args += ' --png --save {0}'.format(self._filename)
         fswebcam = Popen(args, shell=True)
         fswebcam.wait()
 
     def _show_picture(self):
-        self._write_image(ImageMath.eval('convert(img, "1")', img=Image.open('raw.png')))
+        if (1 == self._fb.bits_per_pixel):
+            self._write_image(ImageMath.eval('convert(img, "1")', img=Image.open(self._filename)))
+        else:
+            self._write_image(ImageMath.eval('convert(img, "RGB")', img=Image.open(self._filename)))
```

Comment:
Isn't the image already RGB? `ImageMath.eval()` shouldn't bee needed here.

That should speed things up a bit.